lesson05/assignment/index.html

```diff
@@ -1,15 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 05 - CSS Grid</title>
-    </head>
-    <body>
-        <main class="main">
-          
-        </main>
-    </body>
-</html>
+   <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+      <link rel="stylesheet" href="css/style.css" />
+      <title>Assignment 05</title>
+   </head>
+   <body>
+      <main class="main">
+         <div class="grid">
+            <div class="header"> Header</div>
+            <div class="navbar">
+               <div class="nav-wrap">
+                  <nav class="nav">
+                     <ul class="nav__list">
+                        <li class="nav__item">
+                           <a href="#" class="nav__link">Nav Item</a>
+                        </li>
+                        <li class="nav__item">
+                           <a href="#" class="nav__link">Nav Item</a>
+                        </li>
+                        <li class="nav__item">
+                           <a href="#" class="nav__link">Nav Item</a>
+                        </li>
+                        <li class="nav__item">
+                           <a href="#" class="nav__link">Nav Item</a>
+                        </li>
+                     </ul>
+                  </nav>
+               </div>
+            </div>
+            <div class="sidebar"> Sidebar</div>
+            <div class="main-content"> Main-content</div>
+            <div class="footer"> Footer</div>
+         </div>
+      </main>
```

Comment:
Remember, semantic elements save us from having to use boilerplate classnames. The `main` element doesn't also need a class of "main". The `div class="footer"` element can instead be a `footer` element. The `div class="sidebar"` might be replaced with an `aside` element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).